src/elements/content-preview/__tests__/ContentPreview.test.js

```diff
@@ -1253,4 +1316,37 @@ describe('elements/content-preview/ContentPreview', () => {
             },
         );
     });
+
+    describe('getThumbnail()', () => {
```

Comment:
Should we add a test to ensure that the thumbnail has the properties that we are expecting. I notice we check for null but it might be helpful to test that the thumbnail is going to be returned properly.

Reply:
Hi, for the thumbnail We are testing that the viewer.getThumbnail is called. So for unit testing I think is the way to go. That viewer function will need to unit test if the thumbnail is being returned properly. Also, this function is going to be removed before going GA since we are going to switch the thumbnail provider (not gonna use PDFjs)